media-libs/netpbm/files/netpbm-10.86.21-format-security.patch

```diff
@@ -0,0 +1,18 @@
+Fix compiling with `CFLAGS` `-Werror=format-security`
```

Comment:
have you sent this upstream ?

Reply:
Not yet, but I plan to, together with the postscript bug.  I'll add a comment when it's done.

Reply:
It's been fixed by upstream in advanced *10.91.03*